client/cz-component-dash.html

```diff
@@ -104,57 +104,74 @@
           type: 'Object',
           value: function() { return []; },
         },
-        component: {
+        updateSLO: {
           type: 'Object',
-          value: function() { return {}; },
+          value: function() { return null; },
         },
-        updateSLO: {
+        issueList: {
           type: 'Object',
-          value: function() { return {}; },
+          value: function() { return null; },
+        },
+        issueBreakdown: {
+          type: 'Object',
+          value: function() { return {}; }
         }
       },
 
+      observers: [
+        'updateIssueBreakdown(issueList)',
+      ],
+
       attached: function() {
-        registerSource('cz-config', 'users', function(users) {
+        registerSource('cz-config', 'users', users => {
           this.set('users', users.map(userData => userData['email']));
-        }.bind(this));
-        registerSource('cz-config', 'components', function(components) {
-          this.set('component', {name: components[this.key]});
-          this.updateComponentIssues(this.component)
-        }.bind(this));
-        registerSource('cz-config', 'updateSLO', function(updateSLO) {
+        });
+        registerSource('cz-config', 'updateSLO', updateSLO => {
           this.set('updateSLO', updateSLO);
```

Comment:
I thought we should use this.set() when the property is used in the template otherwise the template is not guaranteed to react to the change. Is this an incorrect assumption?